citus.spec

```diff
@@ -60,6 +60,19 @@ echo %{pginstdir}/lib/%{sname}.so >> installation_files.list
 [[ -f %{buildroot}%{pginstdir}/lib/citus_columnar.so ]] && echo %{pginstdir}/lib/citus_columnar.so >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}-*.sql >> installation_files.list
 echo %{pginstdir}/share/extension/%{sname}.control >> installation_files.list
+# Since files below may be non-existent in some versions, ignoring the error in case of file absence
+[[ -f %{buildroot}%{pginstdir}/share/extension/citus_columnar.control ]] && echo %{pginstdir}/share/extension/citus_columnar.control >> installation_files.list
+columnar_sql_files=(`find %{buildroot}%{pginstdir}/share/extension -maxdepth 1 -name "columnar-*.sql"`)
+if [ ${#columnar_sql_files[@]} -gt 0 ]; then
+    echo %{pginstdir}/share/extension/columnar-*.sql >> installation_files.list
+fi
+
+citus_columnar_sql_files=(`find %{buildroot}%{pginstdir}/share/extension -maxdepth 1 -name "citus_columnar-*.sql"`)
+if [ ${#citus_columnar_sql_files[@]} -gt 0 ]; then
+    echo %{pginstdir}/share/extension/citus_columnar-*.sql >> installation_files.list
+fi
+
+[[ -f %{pginstdir}/share/extension/citus_columnar.control ]] && echo %{pginstdir}/share/extension/citus_columnar.control >> installation_files.list
```

Comment:
why do we have two lines for citus_columnar.control L64 & L75

Reply:
my fault. Removed it.